stable/graylog/templates/secret.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
 {{ include "graylog.metadataLabels" . | indent 4 }}
 type: Opaque
 data:
+  graylog-root-username: {{ .Values.graylog.rootUsername | b64enc | quote }}
```

Comment:
what is the point of putting `graylog-root-username` secret here? It won't load into Graylog configuration.
Did you miss some files?

Reply:
No, I didn't miss any files. This secret will not be utilized by Graylog in anyway. But it is useful if someone wants to utilize a ServiceMonitor with basicAuth. I was planning to make a PR for the serviceMonitor but I didn't have time these days.

Reply:
It will confuse the user cause root username should be added to server.conf and become the default login name.

https://docs.graylog.org/en/3.2/pages/configuration/server.conf.html

Reply:
but it is already [there](https://github.com/helm/charts/blob/f710f21dd13f93cb8e62dacf2de181f0f974e360/stable/graylog/templates/configmap.yaml#L75)

Reply:
@KongZ does this answer your question?